orchestrator/src/handlers/price/traits.rs

```diff
@@ -1,8 +1,10 @@
 use anyhow::Result;
+use verity_dp_ic::verify::types::ProofTypes;
```

Comment:
`ProofTypes` and much of the logic inside of https://github.com/usherlabs/verity-dp/blob/feature/ic-af/ic/libraries/src/verify/types.rs#L81 is relevant to the IC-ADC and not necessarily to "Verity Data Processor".

To clarify, the Verity DP is a generalised framework that enables the development of things like IC-ADC and X NFTs. 
Modules within Verity DP should be re-purposeable between projects.
Even if some Modules are not used between projects, it's meant to be general.
ie. Prove off-chain requests and verify on the IC. 
What that data, or the sources are is relevant to the repo (IC-ADC) implementing Verity DP.